src/integrations/serde.rs

```diff
@@ -1,13 +1,39 @@
 use serde::{de, ser};
 use serde::ser::SerializeMap;
-use std::collections::HashMap;
 use std::fmt;
+use std::collections::HashMap;
 
 use ::{GraphQLError, Value};
 use ast::InputValue;
 use executor::ExecutionError;
 use parser::{ParseError, Spanning, SourcePosition};
 use validation::RuleError;
+use serde_json::Value as Json;
+
+
+impl InputValue {
+    /// Converts serde_json::Value to juniper::InputValue
+    pub fn from_json(json: Json) -> InputValue {
+        match json {
+          Json::Number(num) => {
```

Comment:
I think we've got two options: either add `u64` to `InputValue` and `Value`, or convert it to either `i64` or `f64` if it's larger than `i64::MAX`.

BTW, I think it would be cleaner if this code read like:

```rust
if let Some(num) = num.as_i64() {
    InputValue::int(num)
}
else if let Some(num) = num.as_f64() {
    InputValue::float(num)
}
```